packages/editor-web-component/src/initial-state.tsx

```diff
@@ -0,0 +1,143 @@
+import { SerloEditorProps } from '@serlo/editor'
+
+export type InitialState = SerloEditorProps['initialState']
```

Comment:
really appreciate you putting a default initial state in here, great thought to be able to plug and play